coaches_training.deck.html

```diff
@@ -140,7 +135,7 @@ <h2>Installfest!</h2>
 
 <p>NightCode is pretty easy to install, as it's just an executable JAR. We'll have a couple thumb drives to go around.</p>
 
-<p>Please actually create and run a Quil project, so all dependencies are downloaded and available.</p>
+<p>Please actually create and run a Quil project, so all dependencies are downloaded and available – this has caused problems before!</p>
```

Comment:
I thought we will only be using maria for the next workshop. Then this step is not necessary anymore

Reply:
My plan, if we decide to have students work with maria.cloud, would be to make a separate PR to change all the materials discussing Nightcode.